openapi-specs/sase/mt-monitor/InsightsResources.yaml

```diff
@@ -43,7 +43,7 @@ components:
         alias: {type: string}
         property: {description: "Upgrade list property. See [UpgradeListProperty](/sase/docs/filters#upgradelistproperty).\
             \   \n", example: 'sub_tenant_id, amiversion, current_software_version,
-            last_upgrade_date, next_upgrade_scheduled_str, status, error, next_upgrade_scheduled',
+            last_upgrade_date,next_upgrade_scheduled_str, status, error, next_upgrade_scheduled',
```

Comment:
It appears the leading space before `next_upgrade_scheduled_str` may have been unintentionally removed?